Dockerfile

```
@@ -81,7 +81,7 @@ RUN rapids-mamba-retry install -y \
     gh \
     git \
     jq \
-    "sccache>=0.3.2" \
+    "sccache>=0.5.0" \
```

Running into issues with 0.5.0 -- temporarily pinning this. cc: @ajschmidt8
```suggestion
    "sccache==0.4.2" \
```